core/utils/xml.ts

```diff
@@ -133,8 +138,9 @@ export function textToDom(text: string): Element {
  * @throws if XML doesn't parse.
  */
 export function textToDomDocument(text: string): Document {
-  const oParser = new DOMParser();
-  return oParser.parseFromString(text, 'text/xml');
+  deprecation.warn(
+      'Blockly.utils.xml.textToDomDocument', 'version 10', 'version 11');
```

Comment:
Will our next release be v10?  If we release a 9.4 then this notice will actually be wrong.  🤔

Reply:
yep this can't go in until @maribethb + @rachel-fenichel confirm whether we're doing a v9 or a v10

Reply:
For history: we are moving to v10